src/components/__stories__/simple_nav.tsx

```diff
@@ -0,0 +1,23 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+
+import styled from "styled-components"
+import SimpleNav from "../simple_nav"
+import TextLink from "../text_link"
+
+import * as fonts from "../../assets/fonts"
+
+const LinkContainer = styled.span`
+  position: relative;
+  top: -8px;
+  font-family: ${ fonts.primary.fontFamily };
+`
+
+storiesOf("Simple Nav", SimpleNav)
+  .add("Simple Nav Bar", () => (
+    <SimpleNav>
+      <LinkContainer>
+        <TextLink href="https://www.artsy.net">Back To Artsy</TextLink>
```

Comment:
I added the link here versus in the actual component due to feedback in the [Title Component](https://github.com/artsy/reaction-force/pull/32#discussion_r103759753). I thought it was a similar case.

I added a styled component in order to be able to style the link. I'm wondering if this is the correct approach...?

Also the href is a hard-coded value. I imagine when it's actually used it would be some sort of relative path or something.